### PR TITLE
feat: Display sign-in info on the admin detail & admin table list

### LINKED ISSUE #128: Show current & last login on admin detail page, maybe admin list table as well
Devise provides us some neat information out of the box that would be useful to display for auditing purposes:

https://github.com/codeRIT/hackathon_manager/blob/master/test/dummy/db/schema.rb#L272-L276

* Current & last signed-in time
* Current & last signed-in IP
* Sign in count

### PR DESCRIPTION
**Pull request by dquach217**
Fixes #128

### CODE REVIEW

**Comment by dquach217** [comment]
@sman591 Wondering if you know a solution, but for some reason I'm unable to display the data for the new columns. Kind of confused because nothing should be wrong in theory.

File: app/assets/javascripts/manage/lib/setupDataTables.js [hunk 1]
```diff
@@ -25,6 +25,11 @@ var setupDataTables = function() {
       { orderable: true, data: 'active' },
       { orderable: true, data: 'receive_weekly_report' },
       { orderable: true, data: 'created_at' },
+      { orderable: true, data: 'current_sign_in_at' },
+      { orderable: true, data: 'last_sign_in_at' },
+      { orderable: true, data: 'current_sign_in_ip' },
+      { orderable: true, data: 'last_sign_in_ip' },
+      { orderable: true, data: 'sign_in_count' },
```
Comment (sman591) [inline app/assets/javascripts/manage/lib/setupDataTables.js]: Could you add `visible: false` to each of these? That way they only show up if someone _really_ wants to see them

File: app/datatables/admin_datatable.rb [hunk 2]
```diff
@@ -9,6 +9,11 @@ def view_columns
       active: { source: "User.is_active", searchable: false },
       receive_weekly_report: { source: "User.receive_weekly_report", searchable: false },
       created_at: { source: "User.created_at", searchable: false },
+      current_sign_in_at: { source: "User.current_sign_in_at" },
+      last_sign_in_at: { source: "User.last_sign_in_at" },
```
Comment (sman591) [inline app/datatables/admin_datatable.rb]: Also add `searchable: false` to both of these `_at` fields and `sign_in_count` (date searching isn't very intuitive with data tables)

File: app/views/manage/admins/index.html.haml [hunk 3]
```diff
@@ -11,4 +11,9 @@
         %th Login access
         %th Weekly report
         %th Registered on
+        %th Signed-in
```
Comment (sman591) [inline app/views/manage/admins/index.html.haml]:
```suggestion
        %th Signed-in on
```

File: app/views/manage/admins/index.html.haml [hunk 4]
```diff
@@ -11,4 +11,9 @@
...
+        %th Previous Signed-in
```
Comment (sman591) [inline app/views/manage/admins/index.html.haml]:
```suggestion
        %th Previously signed-in on
```

File: app/views/manage/admins/index.html.haml [hunk 5]
```diff
@@ -11,4 +11,9 @@
...
+        %th IP
```
Comment (sman591) [inline app/views/manage/admins/index.html.haml]:
```suggestion
        %th Signed-in IP
```

File: app/views/manage/admins/index.html.haml [hunk 6]
```diff
@@ -11,4 +11,9 @@
...
+        %th Last IP
```
Comment (sman591) [inline app/views/manage/admins/index.html.haml]:
```suggestion
        %th Previously signed-in IP
```

File: app/views/manage/admins/show.html.haml [hunk 7]
```diff
@@ -36,6 +36,26 @@
...
+    %p
+      %b IP:
+      = @user.current_sign_in_ip == "::1" ? "127.0.0.1" : @user.current_sign_in_ip
+
+    %p
+      %b Last IP:
+      = @user.last_sign_in_ip == "::1" ? "127.0.0.1" : @user.last_sign_in_ip
+
+    %p
+      %b Sign-in Count:
```
Comment (sman591) [inline app/views/manage/admins/show.html.haml]: ^ Could you mirror the wording tweaks from `index.html.haml` here as well?

File: app/datatables/admin_datatable.rb (hunk 2 above)
Comment (dquach217) [reply app/datatables/admin_datatable.rb]: for those `ip` we want to keep them ` searchable: true `?

Comment (sman591) [reply app/datatables/admin_datatable.rb]: Yeah I don't see why not, if someone wants to paste in an IP and see all logins from that IP address they can

File: app/assets/javascripts/manage/lib/setupDataTables.js [hunk 8]
```diff
@@ -25,6 +25,11 @@ var setupDataTables = function() {
       { orderable: true, data: 'active' },
       { orderable: true, data: 'receive_weekly_report' },
       { orderable: true, data: 'created_at' },
+      { orderable: true, data: 'current_sign_in_at' },
```
Comment (sman591) [inline app/assets/javascripts/manage/lib/setupDataTables.js]: `visible: false` here too

**Comment by dquach217** [comment]
@sman591 I finished applying all changes requested

**Pull request merged**